AdobeStockImageAdminUi/view/adminhtml/web/template/grid/toolbar.html

```diff
@@ -4,7 +4,7 @@
  * See COPYING.txt for license details.
  */
 -->
-<div class="admin__data-grid-header" afterRender="$data.setToolbarNode">
+<div class="admin__data-grid-header masonry-main-toolbar" afterRender="$data.setToolbarNode">
```

Comment:
Classes are generally used for styling and are not recommended to use as a selectors in javascript or UI tests.  
I would suggest to use data-role attribute or create a specific attribute which can be used only for tests.